src/components/App.js

```diff
@@ -12,6 +13,15 @@ const SUB = new NchanSubscriber(
   "wss://coderadio-admin.freecodecamp.org/api/live/nowplaying/coderadio"
 );
 
+SUB.on("error", function(err, errDesc) {
+  Sentry.addBreadcrumb({
+    message: "NchanSubscriber error: " + errDesc
+  });
+  // I'm assuming captureException is appropriate here, I'm not sure what type
+  // the first argument has.
+  Sentry.captureException(err);
```

Comment:
I dove into the declarations files for the module and the `exception` parameter is typed as `any`. 

```js
/**
 * Captures an exception event and sends it to Sentry.
 *
 * @param exception An exception-like object.
 * @returns The generated eventId.
 */
export declare function captureException(exception: any, captureContext?: CaptureContext): string;
```

Reply:
Thanks Nick. We can maybe get clearer output, but this should be okay.